lib/Reader/readerir.cpp

```diff
@@ -3183,44 +3187,48 @@ FlowGraphNode *GenIR::fgNodeGetIDom(FlowGraphNode *FgNode) {
   return Idom;
 }
 
-FlowGraphEdgeList *fgNodeGetSuccessorList(FlowGraphNode *FgNode) {
-  FlowGraphEdgeList *FgEdge = new FlowGraphSuccessorEdgeList(FgNode);
-  if (fgEdgeListGetSink(FgEdge) == nullptr) {
-    return nullptr;
+FlowGraphEdgeIterator::FlowGraphEdgeIterator(FlowGraphNode *Fg,
+                                             bool IsSuccessor)
+    : Impl(nullptr) {
+  if (IsSuccessor) {
+    Impl = std::make_unique<FlowGraphSuccessorEdgeIteratorImpl>(Fg);
```

Comment:
I think you need `llvm::make_unique` here (`#include "llvm/STLExtras.h"`) to build with default gcc on Linux, since it's using c++11 and `std::make_unique` is a c++14 feature.  (I know llvm proper needs to do that, I'm not *sure* about LLILC).